andela/wg-justice-league
# #154240249 Create an endpoint to show all infos for an exercise (read-only)

**What does this PR do?**

In this PR, we have a new endpoint _/api/v2/exerciseinfo/_ that allows a user to get all the exercises. This is a read-only endpoint which means only `GET` can be performed on it. In case you want to specify and return  a single exercise you will need to pass in an id in the url.

**Description of Task to be completed?**

Have the following endpoints working

`
GET   /api/v2/exerciseinfo/
`
`
GET   /api/v2/exerciseinfo/{id}
`

**How should this be manually tested?**

- Follow the README.md to setup wger locally and run it on _http://localhost:8000/_
- Ensure you are logged in and click the tab `About this software` and then `Browse the Api`

**What are the relevant pivotal tracker stories?**

#154240249

**Screenshots**

- GET   /api/v2/exerciseinfo/

<img width="920" alt="screen shot 2018-01-19 at 12 03 31" src="https://user-images.githubusercontent.com/19430799/35142932-e8869324-fd10-11e7-9913-94c332167139.png">

- GET   /api/v2/exerciseinfo/{id}

<img width="1032" alt="screen shot 2018-01-19 at 12 01 04" src="https://user-images.githubusercontent.com/19430799/35142801-81210f52-fd10-11e7-8585-7023961afce1.png">
